measures/UpgradeCosts/tests/upgrade_costs_test.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'minitest/autorun'
+
+require_relative '../measure.rb'
+require_relative '../../../resources/hpxml-measures/HPXMLtoOpenStudio/resources/minitest_helper'
+require_relative '../../../resources/hpxml-measures/HPXMLtoOpenStudio/measure'
+
+class UpgradeCostsTest < MiniTest::Test
+  def sample_files_dir
+    return File.join(File.dirname(__FILE__), '..', '..', '..', 'resources', 'hpxml-measures', 'workflow', 'sample_files')
+  end
+
+  def test_single_family_detached
+    cost_multipliers = {
+      'Fixed (1)' => 1,
+      'Wall Area, Above-Grade, Conditioned (ft^2)' => 1200.0,
+      'Wall Area, Above-Grade, Exterior (ft^2)' => 1200.0 + 290.0,
+      'Wall Area, Below-Grade (ft^2)' => 1200.0,
+      'Floor Area, Conditioned (ft^2)' => 2700.0,
+      'Floor Area, Attic (ft^2)' => 1350.0,
+      'Floor Area, Lighting (ft^2)' => 2700.0,
+      'Roof Area (ft^2)' => 1510.0,
+      'Window Area (ft^2)' => 108.0 * 2 + 72.0 * 2,
+      'Door Area (ft^2)' => 40.0 * 2,
+      'Duct Surface Area (ft^2)' => 150.0 + 50.0,
+      'Size, Heating System (kBtu/h)' => 64.0,
+      'Size, Heating Supplemental System (kBtu/h)' => 0.0,
+      'Size, Cooling System (kBtu/h)' => 48.0,
+      'Size, Water Heater (gal)' => 40.0,
+    }
+    _test_cost_multipliers('base.xml', cost_multipliers)
```

Comment:
I would feel _much_ more comfortable with this measure if we could use the [same test buildings](https://github.com/NREL/resstock/blob/develop/measures/SimulationOutputReport/tests/simulation_output_report_test.rb) as were used in the SimulationOutputReport. 
A) Those tests have far better coverage of different situations. 
B) We'd be testing _actual_ HPXML files that ResStock generates, so we'd see for example that the HVAC cost multipliers code won't currently work for ResStock. 
C) We should get the exact same values for most of the cost multipliers.

Reply:
So we'd generate HPXML versions of those test files? Any ideas where that should happen?

Reply:
Yes. Would it make sense to add OSWs to this tests dir and generate the HPXMLs on the fly for each test?

Reply:
That seems like a good idea.